### PR TITLE
Update classifiers to reflect stable

### PR DESCRIPTION
**Pull request by ichard26**
### Description

Even people browsing PyPI using its search filter shall know Black is now stable 😎 

### Checklist - did you ...

- [x] Add a CHANGELOG entry if necessary? -> n/a
- [x] Add / update tests if necessary? -> n/a
- [x] Add new / update outdated documentation? -> n/a

### CODE REVIEW

**Review by felix-hilden — APPROVED** [review]
🎉

**Review by JelleZijlstra — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def find_python_files(base: Path) -> List[Path]:
     },
     test_suite="tests.test_black",
```
Comment (JelleZijlstra) [inline setup.py:115]: Unrelatedly, isn't this wrong now? We have more test files than this.

Comment (cooperlees) [reply setup.py:115]: This is deprecated anyways and does not help pytest. So it can be removed I’m pretty sure.

**Review by cooperlees — APPROVED** [review]
The time has come. Long time since @ambv showed me black working on his laptop @ work in late 2017 or 2018?

**Pull request merged**